### PR TITLE
Minor fixes to listingEditor/listing

### PR DESCRIPTION
**Pull request by joshdanielson**
- fixed some of the spacing issue with the heading
- added a back link to listingEditor for desktop
- minor styling changes in header
- fixed workPhotos to show when empty

### CODE REVIEW

File: app/source/css/components/TilesList.styl [hunk 1]
```diff
@@ -34,6 +34,7 @@
     &.TilesList--openEnd {
         > *:last-child::after {
             border-bottom: none !important;
+            border-right: none !important;
```
Comment (IagoSRL) [inline app/source/css/components/TilesList.styl]: Why this is needed? (the default style for TilesList has not an ":after border-right")

**Comment by IagoSRL** [comment]
Fixed this duped link
screenshot-2017-11-3 edit your massage therapist listing - loconomics

**Comment by IagoSRL** [comment]
And the spacing (in medium size, should not leave empty space at right, to prevent as much as possible to use two lines; at lg size, I keep that empty space with the col-lg-8).

**Comment by IagoSRL** [comment]
Now looks this way
screenshot-2017-11-3 edit your massage therapist listing - loconomics 1

screenshot-2017-11-3 edit your massage therapist listing - loconomics 3
screenshot-2017-11-3 edit your massage therapist listing - loconomics 2

**Comment by joshdanielson** [comment]
@IagoSRL the border-right is used with the alerts on listingEditor. I couldn't figure out how to target them properly in Listing.styl, and figure there may be other places this could be used.

**Comment by joshdanielson** [comment]
@IagoSRL I think the picture and info now look too close together pushed in the middle. I know the other wasn't "perfect" but better than before and better than them being too close together. I think what I'll end up doing with that space is something with the social media icon links and put the publicBio there on desktop when present and push it where it is on mobile. I didn't do this yet as I don't want to waste too much time with it right now. It's in the new issue you created so I will chip away at it.

**Pull request merged**